x-pack/plugins/drilldowns/url_drilldown/public/lib/url_drilldown.tsx

```diff
@@ -90,21 +96,30 @@ export class UrlDrilldown implements Drilldown<Config, ActionContext, ActionFact
   }) => {
     // eslint-disable-next-line react-hooks/rules-of-hooks
     const variables = React.useMemo(() => this.getVariableList(context), [context]);
+
     return (
-      <UrlDrilldownCollectConfig
-        variables={variables}
-        config={config}
-        onConfig={onConfig}
-        syntaxHelpDocsLink={this.deps.getSyntaxHelpDocsLink()}
-        variablesHelpDocsLink={this.deps.getVariablesHelpDocsLink()}
-      />
+      <KibanaContextProvider
+        services={{
+          uiSettings: this.deps.uiSettings,
+        }}
+      >
+        <UrlDrilldownCollectConfig
+          variables={variables}
+          config={config}
+          onConfig={onConfig}
+          syntaxHelpDocsLink={this.deps.getSyntaxHelpDocsLink()}
+          variablesHelpDocsLink={this.deps.getVariablesHelpDocsLink()}
+        />
+      </KibanaContextProvider>
     );
   };
 
   public readonly CollectConfig = reactToUiComponent(this.ReactCollectConfig);
 
   public readonly createConfig = () => ({
-    url: { template: '' },
+    url: {
+      template: 'Example: https://example.com/?{{event.key}}={{event.value}}',
```

Comment:
I think that because of how we don't have a real placeholder, but a "default" URL instead it is not a good idea to have `Example ` prefix 🤔 

Because now our _default_ URL template is invalid

cc @mdefazio

Reply:
I agree with @Dosant, it feels more natural to have a real URL as that is not a placeholder but an example. I've reverted back to have it without "Example: " prefix, unless you have a strong preference @mdefazio.